vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -233,7 +233,9 @@ func (feeder *clusterStateFeeder) InitFromHistoryProvider(historyProvider histor
 				PodID:         podID,
 				ContainerName: containerName,
 			}
-			if err = feeder.clusterState.AddOrUpdateContainer(containerID, nil); err != nil {
+			klog.V(0).InfoS("Adding", "container", containerID)
+			// TODO @jklaw90: pass the container type here
+			if err = feeder.clusterState.AddOrUpdateContainer(containerID, nil, model.ContainerTypeStandard); err != nil {
```

Comment:
the only issue i can see is the prometheus history can't differentiate between init / non init containers. 
i think we'd have to load all pods before this to get this information.